src/kx122/kx122.hpp

```diff
@@ -511,6 +503,9 @@ namespace upm{
       //Device context
       kx122_context m_kx122;
 
+      //Maximum number of samples that can be stored in the buffer of the KX122
+      const static uint MAX_SAMPLES_IN_BUFFER = 681;
```

Comment:
SWIG seems to have trouble with the static member variable.  Since this is private, does it need to be static?

Reply:
If it is not static, it will be added inside the object.
Would a enum value be better ?

Reply:
I think a `#define` will work just fine, we use them throughout and SWIG seems ok with those.

Reply:
Changed it to use a #define